library/x509_create.c

```diff
@@ -37,44 +37,46 @@ typedef struct {
     const char *name;
     size_t name_len;
     const char*oid;
+    int        tag;
```

Comment:
I'd prefer to call this `default_tag` instead of `tag`, as in all cases where we use `MBEDTLS_ASN1_UTF8_STRING`, the other string tags would actually be allowed, too.